sql/core/src/main/java/org/apache/spark/sql/sources/v2/streaming/StreamWriteSupport.java

```diff
@@ -48,7 +48,7 @@
      * @param options the options for the returned data source writer, which is an immutable
      *                case-insensitive string-to-string map.
      */
-    Optional<ContinuousWriter> createContinuousWriter(
+    Optional<StreamWriter> createStreamWriter(
```

Comment:
do we still need to return `Optional`? In which case an implementation should return `None`?

Reply:
You're right, we don't need to. Changed to just StreamWriter.